DESCRIPTION

```diff
@@ -87,10 +87,12 @@ Suggests:
     dygraphs,
     tibble,
     fs,
-    callr (>= 2.0.0)
+    callr (>= 2.0.0),
+    rmarkdown
 SystemRequirements: pandoc (>= 1.12.3) - http://pandoc.org
-URL: https://rmarkdown.rstudio.com
+URL: https://rmarkdown.rstudio.com, https://github.com/rstudio/rmarkdown
 BugReports: https://github.com/rstudio/rmarkdown/issues
 License: GPL-3
-RoxygenNote: 6.1.1
+RoxygenNote: 6.1.1.9000
```

Comment:
That's a superfluous change that shouldn't have been committed.

Reply:
Would you like me to remove? Sorry.

Reply:
I'll do it. No worries. People commit superfluous stuff all the time (e.g. think about how many times you have seen `.DS_Store` in a repo) :)